SU-HSDO/suhumsci
# feat(STN-459): ensure Spotlight works on legacy theme

# READY FOR REVIEW

## Summary
- reverts the change that reverted the Spotlight work (adds all config back in)
- adds scss and css folders to su_humsci theme for the spotlight theme
- adds basic styles for the spotlight theme

## Browser Testing

- TODO:
- ~~get to show up in the hero region~~
- we are going to let H&S do this on a site by site basis now that the config is unlocked for the hero region on basic pages in production due to: https://github.com/SU-HSDO/suhumsci/commit/c665edb166f66b5ac6488c548168f7e8c2b296bb

## Steps to Test
1. `npm i` in the root project directory. Check that `npm run lint:sass` passes and `npm run build` compiles the sass but with no changes.
2. Create a spotlight page with left/right tall/short variations. Check that the design looks good in su_humsci theme as well as traditional and colorful.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
